src/firmware/RC_firmware/RC_firmware.ino

```diff
@@ -0,0 +1,235 @@
+/* Firmware for Elsa
+   Test version
+   Author: Vincent Yuan, Modified: Nick Wu
+     Modified: James Asefa
+   Date: Oct 30, 2016
+*/
+
+#include <SoftwareSerial.h>
+#include <stdlib.h>
+#include <Servo.h>
+
+
+#define TRIM 8
+#define BAUD_RATE 115200
+#define BUFFER_SIZE 6
+// assign 128 to be not moving
+// will be used to hold robot in place while
+// reading from the buffer. Robot will be stopped initially.
+#define UNMAPPED_STOP_SPEED 128
+
+// DRIVING MODIFICATIONS
+// modify FR, FL, BR, BL to change speeds: (+) correpsonds to faster speed, (-) for going reverse
+// F:FRONT, B:BACK, R:RIGHT, L:LEFT e.g. FR = FRONT RIGHT WHEEL.
+const int OFFSET = 1;
+const int joystick_margin = 150;
```

Comment:
All caps naming is fine for constants (I've added that to the README). That being said, by this naming convention, `joystick_margin` should be `JOYSTICK_MARGIN`